rust/stackablectl/src/main.rs

```diff
@@ -16,21 +18,45 @@ async fn main() -> Result<(), Error> {
         .without_time()
         .with_target(false);
 
-    tracing_subscriber::fmt()
-        .with_max_level(match app.log_level {
-            Some(level) => LevelFilter::from_level(level),
-            None => LevelFilter::WARN,
-        })
-        .event_format(format)
-        .pretty()
-        .init();
+    let indicatif_layer = IndicatifLayer::new()
+        .with_progress_style(ProgressStyle::with_template("").unwrap())
+        .with_max_progress_bars(
+            15,
+            Some(
+                ProgressStyle::with_template(
+                    "...and {pending_progress_bars} more processes not shown above.",
+                )
+                .unwrap(),
+            ),
+        );
+
+    let level_filter = match app.log_level {
+        Some(level) => LevelFilter::from_level(level),
+        None => LevelFilter::INFO,
+    };
+
+    if level_filter == LevelFilter::DEBUG {
+        tracing_subscriber::registry()
+            .with(
+                fmt::layer()
+                    .event_format(format)
+                    .pretty()
+                    .with_writer(indicatif_layer.get_stderr_writer()),
+            )
+            .init();
+    } else {
+        tracing_subscriber::registry()
+            .with(level_filter)
+            .with(indicatif_layer)
+            .init();
+    }
 
     // Load env vars from optional .env file
     match dotenv() {
         Ok(_) => (),
         Err(err) => {
             if !err.not_found() {
-                println!("{err}")
+                indicatif_println!("{err}")
```

Comment:
Even though it didn't before, we might want to emit this to stderr:


```suggestion
                indicatif_eprintln!("{err}")
```

Reply:
https://github.com/stackabletech/stackable-cockpit/pull/376/commits/6d80e341512eca4abfc65bc610b112ae5f6fac58